<!DOCTYPE html>
<html><head><title>RuntimeException</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">27 types derived from RuntimeException</div><div class="rA">System.Management.Automation (27)</div><div class="rG" id="System.Management.Automation"><div class="rF"><div class="rN">engine\ExtendedTypeSystemException.cs (1)</div>
<a href="../engine/ExtendedTypeSystemException.cs.html#14"><b>14</b>public class ExtendedTypeSystemException : <i>RuntimeException</i></a>
</div>
<div class="rF"><div class="rN">engine\hostifaces\LocalConnection.cs (1)</div>
<a href="../engine/hostifaces/LocalConnection.cs.html#1504"><b>1504</b>public class RunspaceOpenModuleLoadException : <i>RuntimeException</i></a>
</div>
<div class="rF"><div class="rN">engine\lang\scriptblock.cs (1)</div>
<a href="../engine/lang/scriptblock.cs.html#1324"><b>1324</b>public class ScriptBlockToPowerShellNotSupportedException : <i>RuntimeException</i></a>
</div>
<div class="rF"><div class="rN">engine\MshSecurityException.cs (1)</div>
<a href="../engine/MshSecurityException.cs.html#12"><b>12</b>public class PSSecurityException : <i>RuntimeException</i></a>
</div>
<div class="rF"><div class="rN">engine\NativeCommandProcessor.cs (2)</div>
<a href="../engine/NativeCommandProcessor.cs.html#140"><b>140</b>public sealed class NativeCommandExitException : <i>RuntimeException</i></a>
<a href="../engine/NativeCommandProcessor.cs.html#2310"><b>2310</b>public class RemoteException : <i>RuntimeException</i></a>
</div>
<div class="rF"><div class="rN">engine\regex.cs (1)</div>
<a href="../engine/regex.cs.html#436"><b>436</b>public class WildcardPatternException : <i>RuntimeException</i></a>
</div>
<div class="rF"><div class="rN">engine\remoting\common\remotingexceptions.cs (3)</div>
<a href="../engine/remoting/common/remotingexceptions.cs.html#286"><b>286</b>public class PSRemotingDataStructureException : <i>RuntimeException</i></a>
<a href="../engine/remoting/common/remotingexceptions.cs.html#385"><b>385</b>public class PSRemotingTransportException : <i>RuntimeException</i></a>
<a href="../engine/remoting/common/remotingexceptions.cs.html#683"><b>683</b>public class PSDirectException : <i>RuntimeException</i></a>
</div>
<div class="rF"><div class="rN">engine\TypeTable.cs (1)</div>
<a href="../engine/TypeTable.cs.html#1753"><b>1753</b>public class TypeTableLoadException : <i>RuntimeException</i></a>
</div>
<div class="rF"><div class="rN">FormatAndOutput\common\DisplayDatabase\FormatTable.cs (1)</div>
<a href="../FormatAndOutput/common/DisplayDatabase/FormatTable.cs.html#26"><b>26</b>public class FormatTableLoadException : <i>RuntimeException</i></a>
</div>
<div class="rF"><div class="rN">singleshell\config\MshSnapinLoadException.cs (1)</div>
<a href="../singleshell/config/MshSnapinLoadException.cs.html#23"><b>23</b>public class PSSnapInException : <i>RuntimeException</i></a>
</div>
<div class="rF"><div class="rN">utils\CommandDiscoveryExceptions.cs (2)</div>
<a href="../utils/CommandDiscoveryExceptions.cs.html#15"><b>15</b>public class CommandNotFoundException : <i>RuntimeException</i></a>
<a href="../utils/CommandDiscoveryExceptions.cs.html#185"><b>185</b>public class ScriptRequiresException : <i>RuntimeException</i></a>
</div>
<div class="rF"><div class="rN">utils\CommandProcessorExceptions.cs (1)</div>
<a href="../utils/CommandProcessorExceptions.cs.html#12"><b>12</b>public class ApplicationFailedException : <i>RuntimeException</i></a>
</div>
<div class="rF"><div class="rN">utils\ExecutionExceptions.cs (5)</div>
<a href="../utils/ExecutionExceptions.cs.html#23"><b>23</b>public class CmdletInvocationException : <i>RuntimeException</i></a>
<a href="../utils/ExecutionExceptions.cs.html#322"><b>322</b>public class PipelineStoppedException : <i>RuntimeException</i></a>
<a href="../utils/ExecutionExceptions.cs.html#385"><b>385</b>public class PipelineClosedException : <i>RuntimeException</i></a>
<a href="../utils/ExecutionExceptions.cs.html#448"><b>448</b>public class ActionPreferenceStopException : <i>RuntimeException</i></a>
<a href="../utils/ExecutionExceptions.cs.html#750"><b>750</b>public class RedirectedException : <i>RuntimeException</i></a>
</div>
<div class="rF"><div class="rN">utils\HostInterfacesExceptions.cs (1)</div>
<a href="../utils/HostInterfacesExceptions.cs.html#15"><b>15</b>class HostException : <i>RuntimeException</i></a>
</div>
<div class="rF"><div class="rN">utils\MetadataExceptions.cs (1)</div>
<a href="../utils/MetadataExceptions.cs.html#14"><b>14</b>public class MetadataException : <i>RuntimeException</i></a>
</div>
<div class="rF"><div class="rN">utils\ParameterBinderExceptions.cs (1)</div>
<a href="../utils/ParameterBinderExceptions.cs.html#14"><b>14</b>public class ParameterBindingException : <i>RuntimeException</i></a>
</div>
<div class="rF"><div class="rN">utils\ParserException.cs (1)</div>
<a href="../utils/ParserException.cs.html#15"><b>15</b>public class ParseException : <i>RuntimeException</i></a>
</div>
<div class="rF"><div class="rN">utils\SessionStateExceptions.cs (2)</div>
<a href="../utils/SessionStateExceptions.cs.html#16"><b>16</b>public class ProviderInvocationException : <i>RuntimeException</i></a>
<a href="../utils/SessionStateExceptions.cs.html#392"><b>392</b>public class SessionStateException : <i>RuntimeException</i></a>
</div>
</div>
<div class="rH">39 instantiations of RuntimeException</div><div class="rA">System.Management.Automation (39)</div><div class="rG" id="System.Management.Automation"><div class="rF"><div class="rN">engine\CommandDiscovery.cs (1)</div>
<a href="../engine/CommandDiscovery.cs.html#472"><b>472</b>var ex = new <i>RuntimeException</i>(message);</a>
</div>
<div class="rF"><div class="rN">engine\debugger\debugger.cs (1)</div>
<a href="../engine/debugger/debugger.cs.html#5377"><b>5377</b>new <i>RuntimeException</i>(error),</a>
</div>
<div class="rF"><div class="rN">engine\hostifaces\Command.cs (6)</div>
<a href="../engine/hostifaces/Command.cs.html#523"><b>523</b>throw new <i>RuntimeException</i>(message);</a>
<a href="../engine/hostifaces/Command.cs.html#688"><b>688</b>throw new <i>RuntimeException</i>(</a>
<a href="../engine/hostifaces/Command.cs.html#700"><b>700</b>throw new <i>RuntimeException</i>(</a>
<a href="../engine/hostifaces/Command.cs.html#706"><b>706</b>throw new <i>RuntimeException</i>(</a>
<a href="../engine/hostifaces/Command.cs.html#712"><b>712</b>throw new <i>RuntimeException</i>(</a>
<a href="../engine/hostifaces/Command.cs.html#718"><b>718</b>throw new <i>RuntimeException</i>(</a>
</div>
<div class="rF"><div class="rN">engine\InitialSessionState.cs (1)</div>
<a href="../engine/InitialSessionState.cs.html#3770"><b>3770</b>var ex = new <i>RuntimeException</i>(message);</a>
</div>
<div class="rF"><div class="rN">engine\lang\parserutils.cs (1)</div>
<a href="../engine/lang/parserutils.cs.html#1921"><b>1921</b>e = new <i>RuntimeException</i>(message, innerException);</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\ClientRemotePowerShell.cs (1)</div>
<a href="../engine/remoting/client/ClientRemotePowerShell.cs.html#841"><b>841</b>RuntimeException reason = new <i>RuntimeException</i>(msg);</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\Job.cs (2)</div>
<a href="../engine/remoting/client/Job.cs.html#2260"><b>2260</b>Exception reason = new <i>RuntimeException</i>(msg);</a>
<a href="../engine/remoting/client/Job.cs.html#3376"><b>3376</b>failureException = new <i>RuntimeException</i>(</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\remotingprotocolimplementation.cs (1)</div>
<a href="../engine/remoting/client/remotingprotocolimplementation.cs.html#175"><b>175</b>new <i>RuntimeException</i>(</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\ConnectPSSession.cs (4)</div>
<a href="../engine/remoting/commands/ConnectPSSession.cs.html#608"><b>608</b>ex = new <i>RuntimeException</i>(StringUtil.Format(RemotingErrorIdStrings.CannotFindSessionForConnect,</a>
<a href="../engine/remoting/commands/ConnectPSSession.cs.html#731"><b>731</b>reason = new <i>RuntimeException</i>(</a>
<a href="../engine/remoting/commands/ConnectPSSession.cs.html#737"><b>737</b>reason = new <i>RuntimeException</i>(</a>
<a href="../engine/remoting/commands/ConnectPSSession.cs.html#888"><b>888</b>Exception reason = new <i>RuntimeException</i>(msg);</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\DisconnectPSSession.cs (3)</div>
<a href="../engine/remoting/commands/DisconnectPSSession.cs.html#245"><b>245</b>Exception reason = new <i>RuntimeException</i>(msg);</a>
<a href="../engine/remoting/commands/DisconnectPSSession.cs.html#356"><b>356</b>ErrorRecord errorRecord = new ErrorRecord(new <i>RuntimeException</i>(msg),</a>
<a href="../engine/remoting/commands/DisconnectPSSession.cs.html#504"><b>504</b>Exception reason = new <i>RuntimeException</i>(msg, e);</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\InvokeCommandCommand.cs (1)</div>
<a href="../engine/remoting/commands/InvokeCommandCommand.cs.html#1844"><b>1844</b>RuntimeException reason = new <i>RuntimeException</i>(</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\newrunspacecommand.cs (1)</div>
<a href="../engine/remoting/commands/newrunspacecommand.cs.html#529"><b>529</b>reason = new <i>RuntimeException</i>(this.GetMessage(RemotingErrorIdStrings.RemoteRunspaceOpenUnknownState, state));</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\PSRemotingCmdlet.cs (2)</div>
<a href="../engine/remoting/commands/PSRemotingCmdlet.cs.html#4013"><b>4013</b>Exception reason = new <i>RuntimeException</i>(msg);</a>
<a href="../engine/remoting/commands/PSRemotingCmdlet.cs.html#4046"><b>4046</b>Exception reason = new <i>RuntimeException</i>(msg);</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\PushRunspaceCommand.cs (3)</div>
<a href="../engine/remoting/commands/PushRunspaceCommand.cs.html#328"><b>328</b>new <i>RuntimeException</i>(message), &quot;EnterPSSessionCannotConnectDisconnectedSession&quot;,</a>
<a href="../engine/remoting/commands/PushRunspaceCommand.cs.html#359"><b>359</b>new <i>RuntimeException</i>(message, ex), &quot;EnterPSSessionConnectSessionFailed&quot;,</a>
<a href="../engine/remoting/commands/PushRunspaceCommand.cs.html#419"><b>419</b>new <i>RuntimeException</i>(message), &quot;EnterPSSessionConnectSessionNotAvailable&quot;,</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\ReceivePSSession.cs (1)</div>
<a href="../engine/remoting/commands/ReceivePSSession.cs.html#1026"><b>1026</b>ErrorRecord errorRecord = new ErrorRecord(new <i>RuntimeException</i>(msg, reason),</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\removerunspacecommand.cs (1)</div>
<a href="../engine/remoting/commands/removerunspacecommand.cs.html#160"><b>160</b>Exception reason = new <i>RuntimeException</i>(msg);</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\StartJob.cs (4)</div>
<a href="../engine/remoting/commands/StartJob.cs.html#704"><b>704</b>WriteError(new ErrorRecord(new <i>RuntimeException</i>(message), &quot;StartJobFromDefinitionNamePathInvalidNotFileSystemProvider&quot;,</a>
<a href="../engine/remoting/commands/StartJob.cs.html#716"><b>716</b>WriteError(new ErrorRecord(new <i>RuntimeException</i>(message), &quot;StartJobFromDefinitionNamePathInvalidNotSingle&quot;,</a>
<a href="../engine/remoting/commands/StartJob.cs.html#733"><b>733</b>WriteError(new ErrorRecord(new <i>RuntimeException</i>(message), &quot;StartJobFromDefinitionNameNotFound&quot;,</a>
<a href="../engine/remoting/commands/StartJob.cs.html#742"><b>742</b>WriteError(new ErrorRecord(new <i>RuntimeException</i>(message), &quot;StartJobFromDefinitionNameMoreThanOneMatch&quot;,</a>
</div>
<div class="rF"><div class="rN">engine\runtime\Operations\MiscOps.cs (3)</div>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1995"><b>1995</b>runtimeException = new <i>RuntimeException</i>(er.ToString(), er.Exception, er) { WasThrownFromThrowStatement = true, WasRethrown = rethrow };</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#2013"><b>2013</b>exception = new <i>RuntimeException</i>(message, null);</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#2016"><b>2016</b>runtimeException = new <i>RuntimeException</i>(message, exception, er) { WasThrownFromThrowStatement = true, WasRethrown = rethrow };</a>
</div>
<div class="rF"><div class="rN">utils\FormatAndTypeDataHelper.cs (2)</div>
<a href="../utils/FormatAndTypeDataHelper.cs.html#164"><b>164</b>RuntimeException ex = new <i>RuntimeException</i>(message);</a>
<a href="../utils/FormatAndTypeDataHelper.cs.html#199"><b>199</b>RuntimeException ex = new <i>RuntimeException</i>(message);</a>
</div>
</div>
<div class="rH">263 references to RuntimeException</div><div class="rA">Microsoft.PowerShell.Commands.Utility (16)</div><div class="rG" id="Microsoft.PowerShell.Commands.Utility"><div class="rF"><div class="rN">commands\utility\ImplicitRemotingCommands.cs (4)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#713"><b>713</b>private ErrorRecord GetErrorFromRemoteCommand(string commandName, <i>RuntimeException</i> runtimeException)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#1539"><b>1539</b>catch (<i>RuntimeException</i>)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#1653"><b>1653</b>catch (<i>RuntimeException</i> e)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#1779"><b>1779</b>catch (<i>RuntimeException</i> e)</a>
</div>
<div class="rF"><div class="rN">commands\utility\Import-LocalizedData.cs (1)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Import-LocalizedData.cs.html#231"><b>231</b>catch (<i>RuntimeException</i> e)</a>
</div>
<div class="rF"><div class="rN">commands\utility\ShowCommand\ShowCommand.cs (1)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ShowCommand/ShowCommand.cs.html#436"><b>436</b>catch (<i>RuntimeException</i> e)</a>
</div>
<div class="rF"><div class="rN">commands\utility\Update-TypeData.cs (10)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Update-TypeData.cs.html#378"><b>378</b><i>RuntimeException</i> rte = new(s);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Update-TypeData.cs.html#395"><b>395</b>catch (<i>RuntimeException</i> e)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Update-TypeData.cs.html#502"><b>502</b><i>RuntimeException</i> rte = new(s);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Update-TypeData.cs.html#519"><b>519</b>catch (<i>RuntimeException</i> e)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Update-TypeData.cs.html#860"><b>860</b>catch (<i>RuntimeException</i> ex)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Update-TypeData.cs.html#872"><b>872</b><i>RuntimeException</i> rte = new(s);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Update-TypeData.cs.html#1027"><b>1027</b>catch (<i>RuntimeException</i> e)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Update-TypeData.cs.html#1186"><b>1186</b>catch (<i>RuntimeException</i> ex)</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Update-TypeData.cs.html#1229"><b>1229</b><i>RuntimeException</i> rte = new(s);</a>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/Update-TypeData.cs.html#1246"><b>1246</b>catch (<i>RuntimeException</i> e)</a>
</div>
</div>
<div class="rA">System.Management.Automation (247)</div><div class="rG" id="System.Management.Automation"><div class="rF"><div class="rN">engine\CmdletParameterBinderController.cs (1)</div>
<a href="../engine/CmdletParameterBinderController.cs.html#3702"><b>3702</b>catch (<i>RuntimeException</i> runtimeException)</a>
</div>
<div class="rF"><div class="rN">engine\CommandCompletion\PseudoParameterBinder.cs (1)</div>
<a href="../engine/CommandCompletion/PseudoParameterBinder.cs.html#1199"><b>1199</b>catch (<i>RuntimeException</i>)</a>
</div>
<div class="rF"><div class="rN">engine\CommandDiscovery.cs (1)</div>
<a href="../engine/CommandDiscovery.cs.html#472"><b>472</b><i>var</i> ex = new RuntimeException(message);</a>
</div>
<div class="rF"><div class="rN">engine\CommandProcessor.cs (1)</div>
<a href="../engine/CommandProcessor.cs.html#390"><b>390</b>catch (<i>RuntimeException</i> rte)</a>
</div>
<div class="rF"><div class="rN">engine\CommandProcessorBase.cs (3)</div>
<a href="../engine/CommandProcessorBase.cs.html#808"><b>808</b><i>RuntimeException</i> rte = e as <i>RuntimeException</i>;</a>
<a href="../engine/CommandProcessorBase.cs.html#895"><b>895</b>internal void ManageScriptException(<i>RuntimeException</i> e)</a>
</div>
<div class="rF"><div class="rN">engine\EnumExpressionEvaluator.cs (9)</div>
<a href="../engine/EnumExpressionEvaluator.cs.html#32"><b>32</b>throw InterpreterError.NewInterpreterException(expression, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/EnumExpressionEvaluator.cs.html#40"><b>40</b>throw InterpreterError.NewInterpreterException(expression, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/EnumExpressionEvaluator.cs.html#64"><b>64</b>throw InterpreterError.NewInterpreterException(expression, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/EnumExpressionEvaluator.cs.html#80"><b>80</b>throw InterpreterError.NewInterpreterException(expression, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/EnumExpressionEvaluator.cs.html#486"><b>486</b>throw InterpreterError.NewInterpreterException(input, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/EnumExpressionEvaluator.cs.html#495"><b>495</b>throw InterpreterError.NewInterpreterException(input, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/EnumExpressionEvaluator.cs.html#539"><b>539</b>throw InterpreterError.NewInterpreterException(null, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/EnumExpressionEvaluator.cs.html#549"><b>549</b>throw InterpreterError.NewInterpreterException(null, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/EnumExpressionEvaluator.cs.html#559"><b>559</b>throw InterpreterError.NewInterpreterException(null, typeof(<i>RuntimeException</i>),</a>
</div>
<div class="rF"><div class="rN">engine\EnumMinimumDisambiguation.cs (2)</div>
<a href="../engine/EnumMinimumDisambiguation.cs.html#55"><b>55</b>throw InterpreterError.NewInterpreterException(null, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/EnumMinimumDisambiguation.cs.html#95"><b>95</b>throw InterpreterError.NewInterpreterException(null, typeof(<i>RuntimeException</i>),</a>
</div>
<div class="rF"><div class="rN">engine\ExecutionContext.cs (7)</div>
<a href="../engine/ExecutionContext.cs.html#976"><b>976</b>internal <i>RuntimeException</i> CurrentExceptionBeingHandled { get; set; }</a>
<a href="../engine/ExecutionContext.cs.html#1408"><b>1408</b><i>RuntimeException</i> rte = InterpreterError.NewInterpreterException(null, typeof(<i>RuntimeException</i>), null, errorId, resourceString, arguments);</a>
<a href="../engine/ExecutionContext.cs.html#1438"><b>1438</b><i>RuntimeException</i> rte = InterpreterError.NewInterpreterException(null, typeof(<i>RuntimeException</i>), null, errorId, &quot;{0}&quot;, error);</a>
<a href="../engine/ExecutionContext.cs.html#1468"><b>1468</b><i>var</i> rte = e as <i>RuntimeException</i>;</a>
</div>
<div class="rF"><div class="rN">engine\GetCommandCommand.cs (1)</div>
<a href="../engine/GetCommandCommand.cs.html#1352"><b>1352</b>catch (<i>RuntimeException</i>)</a>
</div>
<div class="rF"><div class="rN">engine\hostifaces\HostUtilities.cs (2)</div>
<a href="../engine/hostifaces/HostUtilities.cs.html#340"><b>340</b>else if (lastError is <i>RuntimeException</i>)</a>
<a href="../engine/hostifaces/HostUtilities.cs.html#342"><b>342</b>lastErrorRecord = ((<i>RuntimeException</i>)lastError).ErrorRecord;</a>
</div>
<div class="rF"><div class="rN">engine\hostifaces\LocalPipeline.cs (2)</div>
<a href="../engine/hostifaces/LocalPipeline.cs.html#608"><b>608</b>catch (<i>RuntimeException</i> ex)</a>
<a href="../engine/hostifaces/LocalPipeline.cs.html#902"><b>902</b>catch (<i>RuntimeException</i>)</a>
</div>
<div class="rF"><div class="rN">engine\hostifaces\Pipeline.cs (2)</div>
<a href="../engine/hostifaces/Pipeline.cs.html#500"><b>500</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;</a>
<a href="../engine/hostifaces/Pipeline.cs.html#568"><b>568</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;</a>
</div>
<div class="rF"><div class="rN">engine\hostifaces\PowerShell.cs (11)</div>
<a href="../engine/hostifaces/PowerShell.cs.html#2280"><b>2280</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;</a>
<a href="../engine/hostifaces/PowerShell.cs.html#2339"><b>2339</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;</a>
<a href="../engine/hostifaces/PowerShell.cs.html#2401"><b>2401</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;</a>
<a href="../engine/hostifaces/PowerShell.cs.html#2460"><b>2460</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;</a>
<a href="../engine/hostifaces/PowerShell.cs.html#2523"><b>2523</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;</a>
<a href="../engine/hostifaces/PowerShell.cs.html#2587"><b>2587</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;</a>
<a href="../engine/hostifaces/PowerShell.cs.html#2654"><b>2654</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;</a>
<a href="../engine/hostifaces/PowerShell.cs.html#2716"><b>2716</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;</a>
<a href="../engine/hostifaces/PowerShell.cs.html#2787"><b>2787</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;</a>
<a href="../engine/hostifaces/PowerShell.cs.html#4011"><b>4011</b>catch (<i>RuntimeException</i>)</a>
<a href="../engine/hostifaces/PowerShell.cs.html#4496"><b>4496</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;</a>
</div>
<div class="rF"><div class="rN">engine\InitialSessionState.cs (1)</div>
<a href="../engine/InitialSessionState.cs.html#3770"><b>3770</b><i>var</i> ex = new RuntimeException(message);</a>
</div>
<div class="rF"><div class="rN">engine\InternalCommands.cs (4)</div>
<a href="../engine/InternalCommands.cs.html#279"><b>279</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;See Pipeline.Invoke.&lt;/exception&gt;</a>
<a href="../engine/InternalCommands.cs.html#300"><b>300</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;See Pipeline.Invoke.&lt;/exception&gt;</a>
<a href="../engine/InternalCommands.cs.html#324"><b>324</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;See Pipeline.Invoke.&lt;/exception&gt;</a>
<a href="../engine/InternalCommands.cs.html#2229"><b>2229</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;See Pipeline.Invoke.&lt;/exception&gt;</a>
</div>
<div class="rF"><div class="rN">engine\lang\interface\PSParseError.cs (1)</div>
<a href="../engine/lang/interface/PSParseError.cs.html#23"><b>23</b>internal PSParseError(<i>RuntimeException</i> rte)</a>
</div>
<div class="rF"><div class="rN">engine\lang\interface\PSParser.cs (2)</div>
<a href="../engine/lang/interface/PSParser.cs.html#140"><b>140</b>/// &lt;exception cref=&quot;System.Management.Automation.<i>RuntimeException</i>&quot;&gt;</a>
<a href="../engine/lang/interface/PSParser.cs.html#167"><b>167</b>/// &lt;exception cref=&quot;System.Management.Automation.<i>RuntimeException</i>&quot;&gt;</a>
</div>
<div class="rF"><div class="rN">engine\lang\parserutils.cs (29)</div>
<a href="../engine/lang/parserutils.cs.html#327"><b>327</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;The result could not be represented as an integer.&lt;/exception&gt;</a>
<a href="../engine/lang/parserutils.cs.html#355"><b>355</b><i>RuntimeException</i> re = new RuntimeException(mice.Message, mice);</a>
<a href="../engine/lang/parserutils.cs.html#372"><b>372</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;An error occurred performing the operation, see inner exception.&lt;/exception&gt;</a>
<a href="../engine/lang/parserutils.cs.html#402"><b>402</b>throw InterpreterError.NewInterpreterException(lval, typeof(<i>RuntimeException</i>), errorPosition,</a>
<a href="../engine/lang/parserutils.cs.html#541"><b>541</b>throw InterpreterError.NewInterpreterException(rval, typeof(<i>RuntimeException</i>), errorPosition,</a>
<a href="../engine/lang/parserutils.cs.html#929"><b>929</b>throw InterpreterError.NewInterpreterException(rval, typeof(<i>RuntimeException</i>), errorPosition,</a>
<a href="../engine/lang/parserutils.cs.html#963"><b>963</b>throw InterpreterError.NewInterpreterExceptionWithInnerException(pattern, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/lang/parserutils.cs.html#1085"><b>1085</b>throw InterpreterError.NewInterpreterException(rval, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/lang/parserutils.cs.html#1126"><b>1126</b>throw InterpreterError.NewInterpreterException(rval, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/lang/parserutils.cs.html#1278"><b>1278</b>catch (<i>RuntimeException</i>)</a>
<a href="../engine/lang/parserutils.cs.html#1292"><b>1292</b>throw InterpreterError.NewInterpreterExceptionWithInnerException(list, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/lang/parserutils.cs.html#1421"><b>1421</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;An error occurred moving to the next element in the enumeration.&lt;/exception&gt;</a>
<a href="../engine/lang/parserutils.cs.html#1433"><b>1433</b>catch (<i>RuntimeException</i>)</a>
<a href="../engine/lang/parserutils.cs.html#1447"><b>1447</b>throw InterpreterError.NewInterpreterExceptionWithInnerException(enumerator, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/lang/parserutils.cs.html#1464"><b>1464</b>catch (<i>RuntimeException</i>)</a>
<a href="../engine/lang/parserutils.cs.html#1478"><b>1478</b>throw InterpreterError.NewInterpreterExceptionWithInnerException(enumerator, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/lang/parserutils.cs.html#1519"><b>1519</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;Wraps the exception returned from the method call.&lt;/exception&gt;</a>
<a href="../engine/lang/parserutils.cs.html#1542"><b>1542</b>throw InterpreterError.NewInterpreterException(methodName, typeof(<i>RuntimeException</i>), errorPosition, &quot;InvokeMethodOnNull&quot;, ParserStrings.InvokeMethodOnNull);</a>
<a href="../engine/lang/parserutils.cs.html#1582"><b>1582</b>throw InterpreterError.NewInterpreterException(methodName, typeof(<i>RuntimeException</i>), errorPosition,</a>
<a href="../engine/lang/parserutils.cs.html#1587"><b>1587</b>throw InterpreterError.NewInterpreterException(methodName, typeof(<i>RuntimeException</i>), errorPosition,</a>
<a href="../engine/lang/parserutils.cs.html#1601"><b>1601</b>throw InterpreterError.NewInterpreterException(methodName, typeof(<i>RuntimeException</i>), errorPosition,</a>
<a href="../engine/lang/parserutils.cs.html#1627"><b>1627</b>catch (<i>RuntimeException</i> rte)</a>
<a href="../engine/lang/parserutils.cs.html#1646"><b>1646</b>throw InterpreterError.NewInterpreterExceptionByMessage(typeof(<i>RuntimeException</i>), errorPosition,</a>
<a href="../engine/lang/parserutils.cs.html#1803"><b>1803</b>internal static <i>RuntimeException</i> NewInterpreterException(object targetObject,</a>
<a href="../engine/lang/parserutils.cs.html#1824"><b>1824</b>internal static <i>RuntimeException</i> NewInterpreterExceptionWithInnerException(object targetObject,</a>
<a href="../engine/lang/parserutils.cs.html#1833"><b>1833</b><i>RuntimeException</i> rte = null;</a>
<a href="../engine/lang/parserutils.cs.html#1899"><b>1899</b>internal static <i>RuntimeException</i> NewInterpreterExceptionByMessage(</a>
<a href="../engine/lang/parserutils.cs.html#1908"><b>1908</b><i>RuntimeException</i> e;</a>
<a href="../engine/lang/parserutils.cs.html#1932"><b>1932</b>private static <i>RuntimeException</i> NewBackupInterpreterException(</a>
</div>
<div class="rF"><div class="rN">engine\lang\scriptblock.cs (8)</div>
<a href="../engine/lang/scriptblock.cs.html#157"><b>157</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;</a>
<a href="../engine/lang/scriptblock.cs.html#218"><b>218</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;</a>
<a href="../engine/lang/scriptblock.cs.html#265"><b>265</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;</a>
<a href="../engine/lang/scriptblock.cs.html#308"><b>308</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;</a>
<a href="../engine/lang/scriptblock.cs.html#395"><b>395</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;Thrown if a script runtime exceptionexception occurred.&lt;/exception&gt;</a>
<a href="../engine/lang/scriptblock.cs.html#516"><b>516</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;Thrown if a script runtime exceptionexception occurred.&lt;/exception&gt;</a>
<a href="../engine/lang/scriptblock.cs.html#858"><b>858</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;A script exception occurred.&lt;/exception&gt;</a>
<a href="../engine/lang/scriptblock.cs.html#916"><b>916</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;A script exception occurred.&lt;/exception&gt;</a>
</div>
<div class="rF"><div class="rN">engine\LanguagePrimitives.cs (3)</div>
<a href="../engine/LanguagePrimitives.cs.html#4008"><b>4008</b><i>RuntimeException</i> rte = InterpreterError.NewInterpreterException(valueToConvert, typeof(<i>RuntimeException</i>), null,</a>
<a href="../engine/LanguagePrimitives.cs.html#4035"><b>4035</b>catch (<i>RuntimeException</i> ex)</a>
</div>
<div class="rF"><div class="rN">engine\Modules\ModuleCmdletBase.cs (8)</div>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#453"><b>453</b>catch (<i>RuntimeException</i> pe)</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#494"><b>494</b>catch (<i>RuntimeException</i> pe)</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#4709"><b>4709</b>typeof(<i>RuntimeException</i>),</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#4728"><b>4728</b>typeof(<i>RuntimeException</i>),</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#4911"><b>4911</b>catch (<i>RuntimeException</i> rte)</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#5712"><b>5712</b>catch (<i>RuntimeException</i> e)</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#5809"><b>5809</b>catch (<i>RuntimeException</i> e)</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#5977"><b>5977</b>catch (<i>RuntimeException</i> e)</a>
</div>
<div class="rF"><div class="rN">engine\Modules\NewModuleCommand.cs (1)</div>
<a href="../engine/Modules/NewModuleCommand.cs.html#210"><b>210</b>catch (<i>RuntimeException</i> e)</a>
</div>
<div class="rF"><div class="rN">engine\Modules\TestModuleManifestCommand.cs (2)</div>
<a href="../engine/Modules/TestModuleManifestCommand.cs.html#88"><b>88</b>throw InterpreterError.NewInterpreterException(_path, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/Modules/TestModuleManifestCommand.cs.html#106"><b>106</b>throw InterpreterError.NewInterpreterException(filePaths, typeof(<i>RuntimeException</i>),</a>
</div>
<div class="rF"><div class="rN">engine\MshCmdlet.cs (3)</div>
<a href="../engine/MshCmdlet.cs.html#683"><b>683</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;Represents a script-level exception.&lt;/exception&gt;</a>
<a href="../engine/MshCmdlet.cs.html#698"><b>698</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;Represents a script-level exception.&lt;/exception&gt;</a>
<a href="../engine/MshCmdlet.cs.html#795"><b>795</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;Represents a script-level exception.&lt;/exception&gt;</a>
</div>
<div class="rF"><div class="rN">engine\MshCommandRuntime.cs (2)</div>
<a href="../engine/MshCommandRuntime.cs.html#2442"><b>2442</b>throw InterpreterError.NewInterpreterException(null, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/MshCommandRuntime.cs.html#2895"><b>2895</b><i>RuntimeException</i>.RetrieveMessage(errorRecord),</a>
</div>
<div class="rF"><div class="rN">engine\MshMemberInfo.cs (3)</div>
<a href="../engine/MshMemberInfo.cs.html#1835"><b>1835</b>catch (<i>RuntimeException</i> e)</a>
<a href="../engine/MshMemberInfo.cs.html#1866"><b>1866</b>catch (<i>RuntimeException</i> e)</a>
<a href="../engine/MshMemberInfo.cs.html#2390"><b>2390</b>catch (<i>RuntimeException</i> e)</a>
</div>
<div class="rF"><div class="rN">engine\NativeCommandProcessor.cs (2)</div>
<a href="../engine/NativeCommandProcessor.cs.html#1308"><b>1308</b>typeof(<i>RuntimeException</i>),</a>
<a href="../engine/NativeCommandProcessor.cs.html#1320"><b>1320</b>typeof(<i>RuntimeException</i>),</a>
</div>
<div class="rF"><div class="rN">engine\parser\Compiler.cs (23)</div>
<a href="../engine/parser/Compiler.cs.html#155"><b>155</b>new Type[] { typeof(ErrorRecord), typeof(<i>RuntimeException</i>) },</a>
<a href="../engine/parser/Compiler.cs.html#517"><b>517</b>typeof(<i>RuntimeException</i>).GetProperty(nameof(<i>RuntimeException</i>.ErrorRecord));</a>
<a href="../engine/parser/Compiler.cs.html#1345"><b>1345</b>typeof(<i>RuntimeException</i>),</a>
<a href="../engine/parser/Compiler.cs.html#1410"><b>1410</b>typeof(<i>RuntimeException</i>),</a>
<a href="../engine/parser/Compiler.cs.html#1514"><b>1514</b>typeof(<i>RuntimeException</i>),</a>
<a href="../engine/parser/Compiler.cs.html#1586"><b>1586</b>typeof(<i>RuntimeException</i>),</a>
<a href="../engine/parser/Compiler.cs.html#1629"><b>1629</b>typeof(<i>RuntimeException</i>),</a>
<a href="../engine/parser/Compiler.cs.html#1668"><b>1668</b>typeof(<i>RuntimeException</i>),</a>
<a href="../engine/parser/Compiler.cs.html#1708"><b>1708</b>typeof(<i>RuntimeException</i>),</a>
<a href="../engine/parser/Compiler.cs.html#1750"><b>1750</b><i>var</i> rte = innerException as <i>RuntimeException</i>;</a>
<a href="../engine/parser/Compiler.cs.html#1755"><b>1755</b>typeof(<i>RuntimeException</i>),</a>
<a href="../engine/parser/Compiler.cs.html#2293"><b>2293</b>catches.Add(Expression.Catch(typeof(<i>RuntimeException</i>), Expression.Block(typeof(object), catchExprs)));</a>
<a href="../engine/parser/Compiler.cs.html#2777"><b>2777</b>typeof(<i>RuntimeException</i>),</a>
<a href="../engine/parser/Compiler.cs.html#2819"><b>2819</b>typeof(<i>RuntimeException</i>),</a>
<a href="../engine/parser/Compiler.cs.html#2859"><b>2859</b>typeof(<i>RuntimeException</i>),</a>
<a href="../engine/parser/Compiler.cs.html#4290"><b>4290</b>return ThrowRuntimeError(typeof(<i>RuntimeException</i>), errorID, resourceString, throwResultType, exceptionArgs);</a>
<a href="../engine/parser/Compiler.cs.html#4335"><b>4335</b>Expression.Constant(typeof(<i>RuntimeException</i>), typeof(Type)),    // exceptionType</a>
<a href="../engine/parser/Compiler.cs.html#5263"><b>5263</b>var rte = NewTemp(typeof(<i>RuntimeException</i>), &quot;rte&quot;);</a>
<a href="../engine/parser/Compiler.cs.html#5264"><b>5264</b>var oldrte = NewTemp(typeof(<i>RuntimeException</i>), &quot;oldrte&quot;);</a>
<a href="../engine/parser/Compiler.cs.html#5339"><b>5339</b>var exception = Expression.Parameter(typeof(<i>RuntimeException</i>));</a>
<a href="../engine/parser/Compiler.cs.html#5407"><b>5407</b>var oldexception = NewTemp(typeof(<i>RuntimeException</i>), &quot;oldrte&quot;);</a>
</div>
<div class="rF"><div class="rN">engine\parser\TypeInferenceVisitor.cs (1)</div>
<a href="../engine/parser/TypeInferenceVisitor.cs.html#799"><b>799</b>catch (<i>RuntimeException</i>)</a>
</div>
<div class="rF"><div class="rN">engine\pipeline.cs (4)</div>
<a href="../engine/pipeline.cs.html#529"><b>529</b>catch (<i>RuntimeException</i> e)</a>
<a href="../engine/pipeline.cs.html#550"><b>550</b><i>var</i> rte = new RuntimeException(message, comException);</a>
<a href="../engine/pipeline.cs.html#686"><b>686</b>catch (<i>RuntimeException</i> e)</a>
<a href="../engine/pipeline.cs.html#707"><b>707</b><i>var</i> rte = new RuntimeException(message, comException);</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\ClientRemotePowerShell.cs (1)</div>
<a href="../engine/remoting/client/ClientRemotePowerShell.cs.html#841"><b>841</b><i>RuntimeException</i> reason = new RuntimeException(msg);</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\Job.cs (1)</div>
<a href="../engine/remoting/client/Job.cs.html#758"><b>758</b>if (!(errorRecord.Exception is <i>RuntimeException</i> runtimeException))</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\JobManager.cs (1)</div>
<a href="../engine/remoting/client/JobManager.cs.html#370"><b>370</b>catch (<i>RuntimeException</i> e)</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\RunspaceRef.cs (1)</div>
<a href="../engine/remoting/client/RunspaceRef.cs.html#107"><b>107</b>catch (<i>RuntimeException</i>)</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\ConnectPSSession.cs (2)</div>
<a href="../engine/remoting/commands/ConnectPSSession.cs.html#551"><b>551</b>catch (<i>RuntimeException</i> e)</a>
<a href="../engine/remoting/commands/ConnectPSSession.cs.html#595"><b>595</b>catch (<i>RuntimeException</i> e)</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\CustomShellCommands.cs (1)</div>
<a href="../engine/remoting/commands/CustomShellCommands.cs.html#831"><b>831</b>catch (<i>RuntimeException</i> rte)</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\EnterPSHostProcessCommand.cs (1)</div>
<a href="../engine/remoting/commands/EnterPSHostProcessCommand.cs.html#250"><b>250</b>catch (<i>RuntimeException</i> e)</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\InvokeCommandCommand.cs (1)</div>
<a href="../engine/remoting/commands/InvokeCommandCommand.cs.html#1844"><b>1844</b><i>RuntimeException</i> reason = new RuntimeException(</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\PSRemotingCmdlet.cs (3)</div>
<a href="../engine/remoting/commands/PSRemotingCmdlet.cs.html#2430"><b>2430</b>catch (<i>RuntimeException</i> rte)</a>
<a href="../engine/remoting/commands/PSRemotingCmdlet.cs.html#2434"><b>2434</b>throw InterpreterError.NewInterpreterException(null, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/remoting/commands/PSRemotingCmdlet.cs.html#3907"><b>3907</b>catch (System.Management.Automation.<i>RuntimeException</i> e)</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\ReceivePSSession.cs (2)</div>
<a href="../engine/remoting/commands/ReceivePSSession.cs.html#440"><b>440</b>catch (System.Management.Automation.<i>RuntimeException</i> e)</a>
<a href="../engine/remoting/commands/ReceivePSSession.cs.html#1160"><b>1160</b>catch (<i>RuntimeException</i> e)</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\TestPSSessionConfigurationFile.cs (3)</div>
<a href="../engine/remoting/commands/TestPSSessionConfigurationFile.cs.html#75"><b>75</b>throw InterpreterError.NewInterpreterException(_path, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/remoting/commands/TestPSSessionConfigurationFile.cs.html#93"><b>93</b>throw InterpreterError.NewInterpreterException(filePaths, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/remoting/commands/TestPSSessionConfigurationFile.cs.html#112"><b>112</b>catch (<i>RuntimeException</i> e)</a>
</div>
<div class="rF"><div class="rN">engine\runtime\Binding\Binders.cs (5)</div>
<a href="../engine/runtime/Binding/Binders.cs.html#2492"><b>2492</b>var exception = Expression.Variable(typeof(<i>RuntimeException</i>), &quot;psBinaryNumericOpException&quot;);</a>
<a href="../engine/runtime/Binding/Binders.cs.html#6457"><b>6457</b>throw InterpreterError.NewInterpreterException(null, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/runtime/Binding/Binders.cs.html#7433"><b>7433</b>throw InterpreterError.NewInterpreterException(methodName, typeof(<i>RuntimeException</i>), null,</a>
<a href="../engine/runtime/Binding/Binders.cs.html#7453"><b>7453</b>throw InterpreterError.NewInterpreterException(methodName, typeof(<i>RuntimeException</i>), null,</a>
<a href="../engine/runtime/Binding/Binders.cs.html#7471"><b>7471</b>throw InterpreterError.NewInterpreterException(memberName, typeof(<i>RuntimeException</i>), null,</a>
</div>
<div class="rF"><div class="rN">engine\runtime\Operations\ArrayOps.cs (4)</div>
<a href="../engine/runtime/Operations/ArrayOps.cs.html#65"><b>65</b>throw InterpreterError.NewInterpreterException(times, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/runtime/Operations/ArrayOps.cs.html#224"><b>224</b>throw InterpreterError.NewInterpreterException(index, typeof(<i>RuntimeException</i>), null,</a>
<a href="../engine/runtime/Operations/ArrayOps.cs.html#228"><b>228</b>throw InterpreterError.NewInterpreterExceptionWithInnerException(index, typeof(<i>RuntimeException</i>), null,</a>
<a href="../engine/runtime/Operations/ArrayOps.cs.html#289"><b>289</b>throw InterpreterError.NewInterpreterException(target, typeof(<i>RuntimeException</i>), null, &quot;CannotIndex&quot;,</a>
</div>
<div class="rF"><div class="rN">engine\runtime\Operations\MiscOps.cs (50)</div>
<a href="../engine/runtime/Operations/MiscOps.cs.html#48"><b>48</b>throw InterpreterError.NewInterpreterException(null, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#53"><b>53</b>throw InterpreterError.NewInterpreterException(null, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#61"><b>61</b>throw InterpreterError.NewInterpreterException(null, typeof(<i>RuntimeException</i>), null,</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#113"><b>113</b>typeof(<i>RuntimeException</i>),</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#143"><b>143</b>catch (<i>RuntimeException</i> rte)</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1196"><b>1196</b>catch (<i>RuntimeException</i> rte)</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1203"><b>1203</b>typeof(<i>RuntimeException</i>), null, &quot;RedirectionFailed&quot;, ParserStrings.RedirectionFailed,</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1283"><b>1283</b>if (!(exception is <i>RuntimeException</i> rte))</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1324"><b>1324</b>throw InterpreterError.NewInterpreterException(hashtable, typeof(<i>RuntimeException</i>), errorExtent,</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1339"><b>1339</b>throw InterpreterError.NewInterpreterException(hashtable, typeof(<i>RuntimeException</i>), errorExtent,</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1476"><b>1476</b>internal static int FindMatchingHandler(MutableTuple tuple, <i>RuntimeException</i> rte, Type[] types, ExecutionContext context)</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1516"><b>1516</b>rte = exceptionToPass as <i>RuntimeException</i>;</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1609"><b>1609</b><i>var</i> rte = exception as <i>RuntimeException</i>;</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1690"><b>1690</b><i>RuntimeException</i> rte)</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1830"><b>1830</b>internal static ActionPreference QueryForAction(<i>RuntimeException</i> rte, string message, ExecutionContext context)</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1902"><b>1902</b>internal static void SetErrorVariables(IScriptExtent extent, <i>RuntimeException</i> rte, ExecutionContext context, Pipe outputPipe)</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1935"><b>1935</b>internal static bool ExceptionCannotBeStoppedContinuedOrIgnored(<i>RuntimeException</i> rte, ExecutionContext context)</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1951"><b>1951</b>internal static bool ReportErrorRecord(IScriptExtent extent, <i>RuntimeException</i> rte, ExecutionContext context)</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1979"><b>1979</b>internal static <i>RuntimeException</i> ConvertToException(object result, IScriptExtent extent, bool rethrow)</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1983"><b>1983</b><i>RuntimeException</i> runtimeException = result as <i>RuntimeException</i>;</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#2023"><b>2023</b>internal static <i>RuntimeException</i> ConvertToRuntimeException(Exception exception, IScriptExtent extent)</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#2025"><b>2025</b><i>RuntimeException</i> runtimeException = exception as <i>RuntimeException</i>;</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#2122"><b>2122</b>throw InterpreterError.NewInterpreterException(typeName, typeof(<i>RuntimeException</i>), errorPos,</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#2145"><b>2145</b>throw InterpreterError.NewInterpreterException(typeName, typeof(<i>RuntimeException</i>), errorPos,</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#2159"><b>2159</b>throw InterpreterError.NewInterpreterException(typeName, typeof(<i>RuntimeException</i>), errorPos,</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#2181"><b>2181</b>throw InterpreterError.NewInterpreterException(rval, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#2204"><b>2204</b>throw InterpreterError.NewInterpreterException(null, typeof(<i>RuntimeException</i>), null,</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#2432"><b>2432</b>throw InterpreterError.NewInterpreterExceptionWithInnerException(pattern, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#2447"><b>2447</b>typeof(<i>RuntimeException</i>), errorExtent, &quot;InvalidFilenameOption&quot;, ParserStrings.InvalidFilenameOption);</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#2460"><b>2460</b>throw InterpreterError.NewInterpreterException(filePath, typeof(<i>RuntimeException</i>), errorExtent,</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#2469"><b>2469</b>throw InterpreterError.NewInterpreterException(filePath, typeof(<i>RuntimeException</i>), errorExtent,</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#2476"><b>2476</b>throw InterpreterError.NewInterpreterException(filePaths, typeof(<i>RuntimeException</i>), errorExtent,</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#2482"><b>2482</b>catch (<i>RuntimeException</i> rte)</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#2814"><b>2814</b>throw InterpreterError.NewInterpreterException(expression, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#2845"><b>2845</b>throw InterpreterError.NewInterpreterException(expression, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#2949"><b>2949</b>throw InterpreterError.NewInterpreterException(null, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#3105"><b>3105</b>throw InterpreterError.NewInterpreterException(null, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#3153"><b>3153</b><i>RuntimeException</i> rte = tie.InnerException as <i>RuntimeException</i>;</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#3195"><b>3195</b>throw InterpreterError.NewInterpreterException(null, typeof(<i>RuntimeException</i>), null,</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#3245"><b>3245</b>catch (<i>RuntimeException</i>)</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#3330"><b>3330</b>catch (<i>RuntimeException</i>)</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#3351"><b>3351</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;An error occurred moving to the next element in the enumeration.&lt;/exception&gt;</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#3363"><b>3363</b>catch (<i>RuntimeException</i>)</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#3377"><b>3377</b>throw InterpreterError.NewInterpreterExceptionWithInnerException(enumerator, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#3393"><b>3393</b>catch (<i>RuntimeException</i>)</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#3407"><b>3407</b>throw InterpreterError.NewInterpreterExceptionWithInnerException(enumerator, typeof(<i>RuntimeException</i>),</a>
</div>
<div class="rF"><div class="rN">engine\runtime\Operations\StringOps.cs (2)</div>
<a href="../engine/runtime/Operations/StringOps.cs.html#46"><b>46</b>throw InterpreterError.NewInterpreterException(times, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/runtime/Operations/StringOps.cs.html#80"><b>80</b>throw InterpreterError.NewInterpreterException(formatString, typeof(<i>RuntimeException</i>),</a>
</div>
<div class="rF"><div class="rN">engine\runtime\Operations\VariableOps.cs (4)</div>
<a href="../engine/runtime/Operations/VariableOps.cs.html#164"><b>164</b>throw InterpreterError.NewInterpreterException(SpecialVariables.AutomaticVariables[tupleIndex], typeof(<i>RuntimeException</i>),</a>
<a href="../engine/runtime/Operations/VariableOps.cs.html#202"><b>202</b>throw InterpreterError.NewInterpreterException(variablePath.UserPath, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/runtime/Operations/VariableOps.cs.html#221"><b>221</b>throw InterpreterError.NewInterpreterException(variablePath, typeof(<i>RuntimeException</i>), null,</a>
<a href="../engine/runtime/Operations/VariableOps.cs.html#316"><b>316</b>throw InterpreterError.NewInterpreterException(null, typeof(<i>RuntimeException</i>),</a>
</div>
<div class="rF"><div class="rN">engine\runtime\ScriptBlockToPowerShell.cs (5)</div>
<a href="../engine/runtime/ScriptBlockToPowerShell.cs.html#361"><b>361</b>catch (<i>RuntimeException</i> rte)</a>
<a href="../engine/runtime/ScriptBlockToPowerShell.cs.html#367"><b>367</b>exceptionType: typeof(<i>RuntimeException</i>),</a>
<a href="../engine/runtime/ScriptBlockToPowerShell.cs.html#539"><b>539</b>throw InterpreterError.NewInterpreterException(null, typeof(<i>RuntimeException</i>),</a>
<a href="../engine/runtime/ScriptBlockToPowerShell.cs.html#562"><b>562</b>catch (<i>RuntimeException</i> rte)</a>
<a href="../engine/runtime/ScriptBlockToPowerShell.cs.html#566"><b>566</b>throw InterpreterError.NewInterpreterException(null, typeof(<i>RuntimeException</i>),</a>
</div>
<div class="rF"><div class="rN">engine\ScriptCommandProcessor.cs (1)</div>
<a href="../engine/ScriptCommandProcessor.cs.html#585"><b>585</b>catch (<i>RuntimeException</i> e)</a>
</div>
<div class="rF"><div class="rN">engine\TypeTable.cs (2)</div>
<a href="../engine/TypeTable.cs.html#4669"><b>4669</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;Throw when there were failures during the update.&lt;/exception&gt;</a>
<a href="../engine/TypeTable.cs.html#4695"><b>4695</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;Throw if there were failures when remove the type.&lt;/exception&gt;</a>
</div>
<div class="rF"><div class="rN">FormatAndOutput\common\Utilities\Mshexpression.cs (1)</div>
<a href="../FormatAndOutput/common/Utilities/Mshexpression.cs.html#344"><b>344</b>catch (<i>RuntimeException</i> e)</a>
</div>
<div class="rF"><div class="rN">help\CommandHelpProvider.cs (1)</div>
<a href="../help/CommandHelpProvider.cs.html#259"><b>259</b>catch (<i>RuntimeException</i>)</a>
</div>
<div class="rF"><div class="rN">utils\FormatAndTypeDataHelper.cs (2)</div>
<a href="../utils/FormatAndTypeDataHelper.cs.html#164"><b>164</b><i>RuntimeException</i> ex = new RuntimeException(message);</a>
<a href="../utils/FormatAndTypeDataHelper.cs.html#199"><b>199</b><i>RuntimeException</i> ex = new RuntimeException(message);</a>
</div>
<div class="rF"><div class="rN">utils\GraphicalHostReflectionWrapper.cs (2)</div>
<a href="../utils/GraphicalHostReflectionWrapper.cs.html#55"><b>55</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;When it was not possible to load Microsoft.PowerShell.GraphicalHost.dlly.&lt;/exception&gt;</a>
<a href="../utils/GraphicalHostReflectionWrapper.cs.html#72"><b>72</b>/// &lt;exception cref=&quot;<i>RuntimeException</i>&quot;&gt;When it was not possible to load Microsoft.PowerShell.GraphicalHost.dlly.&lt;/exception&gt;</a>
</div>
<div class="rF"><div class="rN">utils\PsUtils.cs (1)</div>
<a href="../utils/PsUtils.cs.html#324"><b>324</b>catch (<i>RuntimeException</i> ex)</a>
</div>
<div class="rF"><div class="rN">utils\RuntimeException.cs (2)</div>
<a href="../utils/RuntimeException.cs.html#276"><b>276</b><i>RuntimeException</i> exception = _errorRecord.Exception as <i>RuntimeException</i>;</a>
</div>
<div class="rF"><div class="rN">utils\SessionStateExceptions.cs (8)</div>
<a href="../utils/SessionStateExceptions.cs.html#64"><b>64</b>: base(<i>RuntimeException</i>.RetrieveMessage(innerException), innerException)</a>
<a href="../utils/SessionStateExceptions.cs.html#95"><b>95</b>: base(<i>RuntimeException</i>.RetrieveMessage(errorRecord),</a>
<a href="../utils/SessionStateExceptions.cs.html#96"><b>96</b><i>RuntimeException</i>.RetrieveException(errorRecord))</a>
<a href="../utils/SessionStateExceptions.cs.html#273"><b>273</b>return <i>RuntimeException</i>.RetrieveMessage(innerException);</a>
<a href="../utils/SessionStateExceptions.cs.html#280"><b>280</b>return <i>RuntimeException</i>.RetrieveMessage(innerException);</a>
<a href="../utils/SessionStateExceptions.cs.html#288"><b>288</b>return <i>RuntimeException</i>.RetrieveMessage(innerException);</a>
<a href="../utils/SessionStateExceptions.cs.html#300"><b>300</b><i>RuntimeException</i>.RetrieveMessage(innerException));</a>
<a href="../utils/SessionStateExceptions.cs.html#310"><b>310</b><i>RuntimeException</i>.RetrieveMessage(innerException));</a>
</div>
</div>
</body></html>